src/components/Typography/Typography.style.ts

```diff
@@ -26,113 +26,103 @@ export const typographyWrapper =
     const headline01 = css`
       font-family: ${theme.globals.typography.fontFamily.get('roboto')};
       line-height: ${theme.globals.typography.lineHeight.get('10')};
-      font-size: ${theme.globals.typography.fontSize.get('9')};
+      font-size: ${theme.globals.typography.fontSize.get('10')};
       font-weight: ${theme.globals.typography.fontWeight.get('bold')};
-      letter-spacing: ${theme.globals.typography.letterSpacing.get('0')};
-      margin-block-start: 1rem;
-      margin-block-end: 1rem;
+      letter-spacing: ${theme.globals.typography.letterSpacing.get('0')}px;
```

Comment:
based on the `theme.globals.typography.letterSpacing.get` implementation, letterSpacing values from tokens are expected to be in px and then converted to rem. So `theme.globals.typography.letterSpacing.get('0')` should return rem, without the need of a px suffix.

By checking `src/theme/globals/constants/letterSpacing.ts` I see that the values are in a wrong and not parsable format (e.g. letterSpacing[2] = .25), that may have something to do with the fact that `px` suffix is needed here. 

We should revisit the letterSpacing tokens and how they are imported from Figma and parsed by the codebase in order to not need suffixes with px or rem.

Thoughts on this?